projects/packages/my-jetpack/_inc/hooks/use-notification-watcher/use-paid-plan-needs-plugin-install-activation-notice.tsx

```diff
@@ -282,11 +292,13 @@ const usePaidPlanNeedsPluginInstallActivationNotice = ( redBubbleAlerts: RedBubb
 			priority: NOTICE_PRIORITY_MEDIUM + ( isInstallingOrActivating ? 1 : 0 ),
 		};
 
-		setNotice( {
-			title: noticeTitle,
-			message: noticeContent,
-			options: noticeOptions,
-		} );
+		if ( ! isLoading && ! checkForCookie( `${ planSlug }--plugins_needing_installed_dismissed` ) ) {
```

Comment:
We are checking here for cookie as well, since the REST API call doesn't send over the cookies when retrieving the red bubble slugs from the backend. Instead of increasing security risk by sending all cookies along with the request, I thought we could just check for it here as well since it's not much added code